### ISSUE TITLE
🎉 Outreachy spring 2022: Applicants tasks/contributions

### ISSUE THREAD

**Issue by BatoolMM** [Open-Science-Community-Saudi-Arabia/localisation#1]
### 🎊 Welcome Outreachy 2022 Contributors! We're excited that you're here 🎉

>This ticket will serve as the central point tracking the contributions of perspective Outreachy (spring 2022 cohort) applicants.

Outreachy provides paid remote, three-month internships to support diversity in Free and Open Source Software (FOSS). For more information about what Outreachy is, follow [this link](https://www.outreachy.org/).

**Outreachy internships are:**

💵 Paid - $7,000 USD total internship stipend

🌍 Remote - both interns and mentors work remotely

🕰 3 months - internships run May to August.

For those interested, this process is described [here](https://www.outreachy.org/apply/make-contributions/).

> We ask all outreachy candidates to work **on a minimum of 3 projects** simultaneously so that they are not completely relying on their selection in OSCSA if we end up not selecting them!

## Summary

Open Science Community Saudi Arabia (OSCSA) aims to introduce open educational resources (OERs) for Open Science in Arabic to facilitate learning of Open science practices to novice learners. Therefore, we work with other Open Science communities such as the Turing Way and Open Life Science to localise their open educational resources (OER) using [Crowdin](https://crowdin.com/), GitHub and Zenodo.

We use GitHub as a version control system for all the translations and to acknowledge all contributors. Crowdin is used as a translation management system (or TMS) to make contributions smooth. Any changes made to Crowdin will be reflected on GitHub using automation and continuous integration (CI). Crowdin provides Translation Memory (TM), Glossary, Machine Translation (MT) and QA checks.

In Outreachy spring 2022, we are accepting applicants to localise specific chapters of  [_The Turing Way_](https://the-turing-way.netlify.app/welcome.html). The Turing Way is a handbook to reproducible, ethical and collaborative data science. It supports a diverse community of contributors to make data science accessible, comprehensible and effective for everyone. You can find more information about it, in [the TW repository](https://github.com/alan-turing-institute/the-turing-way).

Successful candidates will improve the localisation process by adding documentation in Arabic, enriching the glossary, moderating the translation, using screenshots to improve the translation and collecting feedback.

### How to start?

- Join OSCSA community [here](https://form.jotform.com/220362171812446)

- You will need to create an account in Crowdin and join [our project here](https://turingway.crowdin.com/u/signup?hash=caa0e3045741565a01837bdb3245c03294). Once you join, you have to comment on this issue, so we follow your work in Crowdin.
- Navigate to the Project home and click on Arabic 
<img width="1418" alt="Screenshot 2022-03-28 at 19 55 05" src="https://user-images.githubusercontent.com/53487593/160449190-4a5e7421-7171-47d5-9c66-d7bd29b905d8.png">

- Once you click on Arabic, you will be directed to this window which contains all the files that are inside the Turing Way book 
<img width="1416" alt="Screenshot 2022-03-28 at 19 55 26" src="https://user-images.githubusercontent.com/53487593/160449198-05aee1ba-cdfb-4e9c-9f41-d17ca275a229.png">

- Scroll down until you reach `reproducibale_reserach` and then to `open` . Translate one the topics in this [task](https://turingway.crowdin.com/u/projects/1/tasks/18) in Crowdin.
<img width="1440" alt="Screenshot 2022-03-28 at 20 02 34" src="https://user-images.githubusercontent.com/53487593/160450847-a66efcf2-17d7-4e4e-b144-55ef7428f244.png">

- Choose any of the markdown files inside and click on three dots (⋮) select **Go to editor** 
<img width="1421" alt="Screenshot 2022-03-28 at 20 04 03" src="https://user-images.githubusercontent.com/53487593/160450765-49765d54-8d26-419a-bf5b-de20c2e9c9de.png">

- This way, it will open the crowdin editor where you will so all the translations! All the translation was done using crowdin machine translation (similar to google translation). None of it was done manually, so all the translation needs to be re-translated and then proofread (Approved). Chose only one single file and read it from [the TW book](https://the-turing-way.netlify.app/) before starting the translation - and start from the top of the file (title).
<img width="1433" alt="Screenshot 2022-03-28 at 19 57 07" src="https://user-images.githubusercontent.com/53487593/160449211-678c3a71-b184-4d65-930e-b3ef1db4b02b.png">

- Follow up the steps of how to use Crowdin Editor [here](https://github.com/Open-Science-Community-Saudi-Arabia/ols3-cohort-talks-and-transcripts/blob/main/CONTRIBUTING%E2%80%8B.md).

Using the Main menu in the top left corner, you can switch between files for translations, change translation language, contact manager, switch the view, get help or quit the editor. For more information, follow [the docs here](https://support.crowdin.com/online-editor/). If you have questions, please add them to this issue!

> You need only to translate one small file with no more than 300 words. You don't only translate strings, but you also have to add terms to [the glossary](https://support.crowdin.com/glossary/). 

### Glossary

The **Glossary** is supposed to contain **terminology that has a purpose of explaining some specific terms** or the ones often used in the project to be translated properly and consistently such as (e.g. open science, open access, open hardware, ...etc). It's not supposed to include names of people, books, ...etc. 

**You should add a specific term add to the glossary if:**
1- It's commonly used in the project, but translators can translate it differently. For example, open access can be translated الوصول الحر أو الوصول المفتوح أو النفاذ الحر  but we want to keep consistency throughout the project when it's translated by many contributors - so they can use the same translation, not multiple ones.
2- Some English terminology doesn't exist in Arabic. Once you translate them, you include them in Glossary (e.g. Git, GitHub, repository,..etc).

The Glossary should be consistent in style. The Arabic translation should contain the Arabic translation followed on the left the English terms between brackets with spaces between brackets. For example, the Arabic for Open access should be (Open Access) الوصول الحر with space between brackets and the Arabic word.

**If you have questions about the glossary،, ask under this issue.**

### Submission

Once you finish a very small part of the translation, submit a PR for a markdown file called `NAME_docs.md` in `/docs` in this repo to include brief documentation in **Arabic** ( around 500 words or more using Markdown) about the translation process of the project in Crowdin. This should include (Getting started with Crowdin, tips, ..etc) using images and GIFs. 

**If you have any questions throughout the process, add your question to this issue!**

- [Record in-progress contributions on the Outreachy website](https://www.outreachy.org/apply/project-selection/). You'll need to [go to the community landing page linked from the projects listing page](https://www.outreachy.org/apply/project-selection/). Then click the 'Record Contribution' button. Add links and descriptions to your contributions and the project timeline before the application deadline. You can always go back and edit your recorded contributions. A contribution doesn't have to be merged or accepted to be recorded.

### Tasks for Intern Selection:
- Complete the translation of one of the landing pages of the Guide for Reproducible Research (e.g. [Overview of Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/overview.html), [Open Research](https://the-turing-way.netlify.app/reproducible-research/open.html), [Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html), ..etc).
- Open PR in this repository to include brief documentation in **Arabic** (400 words or more) about the translation process of the project in Crowdin.

### Tasks during the internship
- Expands the Glossary and ensure consistency in the Translation
- Collect feedback from the community members about the translation workflow.
- Translate 6 parts/sub-chapters from the Guide for Reproducible Research.
- Review other contributions in the Translation 
- Deploy the book with the 6 translated chapters 
- Provide documentation about the Workflow in Arabic.

### Meetings

We will lead a video call in Elements to answer any questions you have. This could be a problem using Crowdin editor or understanding any of the tasks in the internship. This will be on **Friday, the 1st of April at 10 am UTC**. Follow [this link](https://matrix.to/#/#oscsa:matrix.org) for invitation to Element!

### Assessing Candidates

- Candidate should be comfortable collaborating through GitHub
- Candidates should complete all tasks listed for intern selection.

### Resources
- Some documentation about the localisation of _The Turing Way_ can be found [here](https://deploy-preview-2202--the-turing-way.netlify.app/community-handbook/translation.html).
- More information about the workflow is presented [in this talk in Creative Commons Summit 2021](https://zenodo.org/record/5521150#.YjjYpZpBy3I).

**Comment by oluwaseunlijoka** [comment]
Hello everyone,  My name is Oluwaseun, I just joined Crowdin.

**Comment by BatoolMM** [comment]
Hi @oluwaseunlijoka, you should be able to join the project through this link: https://turingway.crowdin.com/u/signup?hash=3d0624a322db57a403a7ece3ccce900a24.

Let me know if you have a problem accessing it!

**Comment by oluwaseunlijoka** [comment]
Hello @BatoolMM, Thank you, I just joined now. I guess you are mentor now. Please can lead me through how I can start, I am from https://www.outreachy.org/. Thank you

**Comment by BatoolMM** [comment]
@oluwaseunlijoka I added a few screenshots and details - let me know here if you struggle with anything!

**Comment by ThanaMeri** [comment]
Hello @BatoolMM , my name is Thana. I just joined the project by following the link you provided.

**Comment by BatoolMM** [comment]
Hi, Thana - you've been added to Crowdin, feel free to follow the steps and let me know if you have any questions!

**Comment by ahmedEid1** [comment]
Hello @[BatoolMM](https://github.com/BatoolMM), I am Ahmed an engineering student from Egypt, and an outreachy applicant for this spring.
I am very existed to help increase the science martials available in Arabic to make learning and getting into the science field easier for any Arabic speaker.

**Comment by ahmedEid1** [comment]
> `you have to comment on this issue, so we can promote you to Manager in Crowdin.`

I just joined the project, do i have to wait to be a manger before starting to contribute ?

**Comment by BatoolMM** [comment]
Hi Ahmed, thank you for joining - you don't have to wait, you should be able to use Crowdin now. Let me know if you have any questions about crowdin.

**Comment by Nizahra2038** [comment]
Hi @BatoolMM my name is Fatima I am oureachy applicant I have joined the crowdin what next

**Comment by nancyalaswad90** [comment]
.

hi @BatoolMM 


I added [issue here](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/2) ,please review and advice

**Comment by AsmaKacem1** [comment]
Hello @BatoolMM , My name is Asma. I'm from Tunisia it's a small country in the north of Africa. I'm a Telecommunication engineering student and an Outreachy applicant. I am
excited to be contributing to this project because I love my native language Arabic and make it easier for the Arabic speaker to learn.  I just joined the project by following the link you provided.

**Comment by BatoolMM** [comment]
Hi @AsmaKacem1 @nancyalaswad90 @Nizahra2038, thank you for joining - please follow the steps in this issue - if any of you have any questions, please add them here under this issue (don't open a new issue). [Here](https://support.crowdin.com/enterprise/getting-started-for-volunteers/) is more info on how to use Crowdin Editor.

**Comment by Boutainaelatbaoui** [comment]
Hello @BatoolMM, I am Boutaina from Morocco. I am an outreachy applicant. I have joined crowdin, and I am super excited to contribute to your organization and be a part of your community.

**Comment by BatoolMM** [comment]
Hi Boutaina! Let me know if you struggle with anything in Crowdin.

**Comment by BatoolMM** [comment]
We will hold a video call in Elements to answer any questions you have. This could be a problem using Crowdin editor or understanding any of the tasks in the internship. This will be on Friday, the 1st of April at 10 am UTC. Follow [this link](https://matrix.to/#/#oscsa:matrix.org) for invitation to Element!

**Comment by Boutainaelatbaoui** [comment]
> Hi Boutaina! Let me know if you struggle with anything in Crowdin.

I'm trying to click on 'go to the editor', but I can't find it. Could you please help me?

![outreachy](https://user-images.githubusercontent.com/56041419/160601746-2cc8ed08-dd68-487d-a953-96bd0cdba2f9.PNG)

**Comment by mejri1337** [comment]
Hi @BatoolMM , I'm Alaa, an aspiring Outreachy intern, looking forward to be working with you.

**Comment by BatoolMM** [comment]
Thank you @Boutainaelatbaoui for the question. If **Go to editor** isn't visible, choose **Proofreading** and you will end up with something similar to this screenshot.

<img width="1428" alt="Screenshot 2022-03-29 at 20 08 29" src="https://user-images.githubusercontent.com/53487593/160667269-3c50225f-485f-44d8-af92-92630e93b17e.png">

You’ll see the list of source strings and their translations that you can approve or correct. The strings with the QA checks issues will be highlighted in this mode. You’ll also see the strings that were translated with the help of Translation Memory or Machine Translation engine. You can still change the complete translation. Let me know if this works for you! 

Hi @amj1337, you are welcome to go through the steps and post your questions here!

**Comment by BatoolMM** [comment]
All individuals who commented on this issue are granted Manager access so they can get access to all features of the Crowdin editor! Let me know if you still don't see **Go to editor** option.

**Comment by nancyalaswad90** [comment]
.
Thanks for your explanation above @BatoolMM 
 
I made some script , will you please give your opinion about it

.

**Comment by Rufaida98** [comment]
@BatoolMM Hello mentor
My name is Rufaida an Outreachy applicant and i want to contribute to this project

**Comment by Rufaida98** [comment]
@BatoolMM i oppened the crowdin.should i choose a file and start translating and add it over here?.please guide me

**Comment by AsmaKacem1** [comment]
Hello,
me too I made some script, could you please verify if there is something I can improve @BatoolMM .
 
Could you please explain to us about the video of OLS2 and  OLS1?

**Comment by BatoolMM** [comment]
Thank you @AsmaKacem1 for adding the translation. Please work in one single markdown file and completely finish it before moving to another file. Can you kindly focus on translating`open-hardware.md` which you started working on? 

I see that you also added a few terms to the Glossary. That's really good - but some terms like *def* are repeated with two different meanings. If you are adding the English word between brackets, please keep space between the bracket and the word - e.g. GitHub : (GitHub) منصة جيت هب - **not** (GitHub)جيتهاب. With space between brackets and the word.

Once you complete a file or a couple of paragraphs from the same file, I'll add comments in Crowdin for you (like below). 

<img width="1429" alt="Screenshot 2022-03-31 at 01 22 51" src="https://user-images.githubusercontent.com/53487593/160940952-e4127c71-9730-4990-a9be-d9f4d9b10827.png">

@nancyalaswad90 Your translation is good, I added some comments on Crowdin - please go through them. You should be able to access the comments tab in Crowdin similar to the screenshot above **(on the right)**!

Let me know if you have any questions!

**Comment by nancyalaswad90** [comment]
Thanks @BatoolMM for your feedback 

I will do the update and send you

**Comment by AsmaKacem1** [comment]
Thank you @BatoolMM . I took these notes and I will do the necessary change you mentioned.

**Comment by nancyalaswad90** [comment]
Thanks @BatoolMM for your  explanation comments 

Just finished some updates  in my translation and added a few terms to the Glossary.  , will you please give your opinion about it.

**Comment by BatoolMM** [comment]
In terms of the **glossary**. The **Glossary** is supposed to contain **terminology that has a purpose of explaining some specific terms** or the ones often used in the project to be translated consistently and adequately, such as (e.g. open science, open access, open hardware, ...etc). It's not supposed to include names of initiatives. 

There are some issues in the **Glossay** below: 

![Screenshot 2022-03-31 at 16 00 13](https://user-images.githubusercontent.com/53487593/161060508-415f3d1a-0979-4c8b-b98c-b27601c2dfe2.png)

- All terms should have been filled in the 1st column (English) since the book is written in English but `GitHub` isn't filled in the 1st column. 
- It's not consistent in style. The Arabic translation should contain the Arabic translation followed **on the left** the English terms between brackets with spaces between brackets. For example, the Arabic for Open access should be ` (Open Access) الوصول الحر ` with space between brackets and the Arabic word. 
- Budapest open access initiative, def, open by default,  isn't supposed to be on the Glossary.
- Please make sure the style of the Glossay is consistent.

**Comment by BatoolMM** [comment]
I added comments for translations made so far by @Boutainaelatbaoui @nancyalaswad90 @AsmaKacem1 in Crowdin. A very important point is that some seem to translate individual sentences without reading the sub-chapter or understanding the context. Please do read the sub-chapter from [the Turing Way book here](https://the-turing-way.netlify.app/welcome.html) and understand what the book is discussing - then choose a topic/subchapter and understand it before you translate it and start the translation from the title of the subchapter, not the middle of the subchapter. 

If anything isn't clear, please feel free to ask questions here!

**Comment by AsmaKacem1** [comment]
Thank you so much @BatoolMM for your feedback.
 I have a question please, when we can add a word to the Glossary and when not?

**Comment by BatoolMM** [comment]
A good question @AsmaKacem1! You should complete reading the sub-chapter to know what is the common words, terminology, concepts and which one fits to be added to the Glossary. You add a specific word add to the glossary if:
- It's commonly used in the project but it can be translated differently by translators. For example, open access can be translated الوصول الحر أو الوصول المفتوح أو النفاذ الحر .
> but we want to keep consistency throughout the project when it's translated by many contributors - so they can use the same translation, not multiple ones.
-  There is some English terminology that doesn't exist in Arabic. Once you translate them, you include them in Glossary.

**The takeaway message, you need to read the sub-chapter before you translate it.**

**Comment by BatoolMM** [comment]
> for example when we found this sentence: [{term}<0>def<1></0>] we should change it to: [{مصطلح} <0> تعريف <1> </0>] ? or not

You don't translate this at all. It's not part of the sentence. It's a syntax `[{term}<0>def<1></0>]`. If you read the chapter from [the TW book](https://the-turing-way.netlify.app/) not from the crowdin editor, you'll realise this isn't part of the sentence. Let me know if this is still not clear!

**Comment by AsmaKacem1** [comment]
Thank you @BatoolMM.
I will read the sub-chapter and the chapter from [the TW book](https://the-turing-way.netlify.app/)  to avoid such mistakes

**Comment by Boutainaelatbaoui** [comment]
Thank you very much @BatoolMM for your feedback.
I will read the sub-chapter and update my translation and words in the Glossary.

**Comment by mejri1337** [comment]
Hi @BatoolMM , is proofreading an already 100% translated file, editing it's translation and approving translation considered as a part of the below task ?
Or should we translate files that are not 100% translated ?

> Complete the translation of one of the landing pages of the Guide for Reproducible Research (e.g. [Overview of Reproducible Research](https://the-turing-way.netlify.app/reproducible-research/overview.html), [Open Research](https://the-turing-way.netlify.app/reproducible-research/open.html), [Research Data Management](https://the-turing-way.netlify.app/reproducible-research/rdm.html), ..etc).

**Comment by BatoolMM** [comment]
Hi @amj1337, this is an excellent question. All the translation was done using crowdin machine translation (similar to google translation). None of it was done manually, so all the translation needs to be re-translated and then proofread (Approved). Chose only one single file and read it from [the TW book](https://the-turing-way.netlify.app/) before starting the translation - and start from the top of the file (title). Please avoid translating random sentences in the middle of the file.  Don't hesitate to ask if you have any other questions!

**Comment by BatoolMM** [comment]
I updated the issue above with more instructions!

**Comment by ThanaMeri** [comment]
are we having a video call on Element now?

**Comment by BatoolMM** [comment]
@ThanaMeri We just finished the call now! If you have any questions, feel free to ask in this issue!

**Comment by BatoolMM** [comment]
We went through [these slides](https://docs.google.com/presentation/d/1vwKgF2Z7mg9SDphMur5HEW0pBaP0AJgMMfbILDQQR28/edit?usp=sharing) and a demo of Crowdin.

**Comment by Boutainaelatbaoui** [comment]
@BatoolMM Is there a recording of the meeting? I've been trying to attend the entire call, but I had a bad internet connection this morning.

**Comment by BatoolMM** [comment]
> @BatoolMM Is there a recording of the meeting? I was trying to join the call, but I had a bad internet connection.

We didn't record the call - so everyone can be comfortable asking questions!

**Comment by nancyalaswad90** [comment]
Dear @BatoolMM 

I make Glossary  such as (GitHub) جيت هاب in the image below and I add the description after making some search from site 

.

![work](https://user-images.githubusercontent.com/36210723/161257843-aa55adf9-9151-4ba2-a401-34f4cf590b21.PNG)


.


> **But I notice that if `any other member` make save to the change `I completely did it by myself` , the system remove my name from the work without mention my work in it** 




## Also other Glossary  from  `other member` (( With all respect to my great team )) ,are added and change the Glossary I made 


### The idea is any one make any change he / she erase the previous peer work 


.


![memeber](https://user-images.githubusercontent.com/36210723/161259457-6e1057e6-7a37-4277-b039-d400f3a34772.PNG)

**Comment by BatoolMM** [comment]
Hi @nancyalaswad90, 

We discussed this issue in the call. The problem is that there's **space for improvement** for some of the translations in the Glossary (and some need correction). These terms come up in other files that are translated by the other team members. If that happens, everyone is allowed to change/improve the translation so it fits the context better. However, if anyone changed the terminology in the Glossay, he/she should add a comment in **the Crowdin editor** that he/she changed a term in the Glossary. 

Having said that, I can still track the original translation if it was created by a different team member from another tab, that's doesn't erase anyone's work - so don't worry about that. Let me know if you have any concerns. 

Please be kind to each other!

**Comment by nancyalaswad90** [comment]
Sure , I will be kind to other member 

> **As I mentioned above (( With all respect to my great team ))**

**Comment by BatoolMM** [comment]
I added a table to organise the translation, based on your feedback - so you can pick a chapter under your name and work on it without worrying that someone else might change it. [Here is the Table](https://docs.google.com/document/d/1u7fbr7XN2twbQAJ4laBvOu0BbXepqdRh42G-KdUsVOI/edit
) with some recommended files/chapters. 
You don't have to use a chapter from the list - you can add any chapter/file that you are translating with your name.

**Comment by nancyalaswad90** [comment]
@BatoolMM  Thank you so much for your quick feedback and for create this idea

I did some changes , I wish you provide me with your feedback 

.

> ![Capture](https://user-images.githubusercontent.com/36210723/161366670-f81bb600-3494-4d14-9e02-f7d9f2b1f09b.PNG)

**Comment by mejri1337** [comment]
Hi @BatoolMM when reading the book I came across words that look similar in their general meaning like "Reproducible" and "Replicable" but they were defined to have different meanings in the book, is it right for those words to be added to the Glossary with their definitions from the Turing way book itself ? even thought they are not defined in the file that I chose to translate but some of them are used in it.

**Comment by BatoolMM** [comment]
Thank you @amj1337 for this question 

> when reading the book I came across words that look similar in their general meaning like "Reproducible" and "Replicable" but they were defined to have different meanings in the book, is it right for those words to be added to the Glossary with their definitions from the Turing way book itself? even though they are not defined in the file that I chose to translate but some of them are used in it.

Absolutely!

**Comment by nancyalaswad90** [comment]
Dear @BatoolMM 

**Please advise** , what should I do now ? and what  is the assessment for my previous updates ?

**Comment by BatoolMM** [comment]
> Dear @BatoolMM
> 
> **Please advise** , what should I do now ? and what is the assessment for my previous updates ?

Done!

**Comment by nancyalaswad90** [comment]
@BatoolMM Thank you  for your quick feedback 


I made some changes , I wish you provide me with your feedback

**Comment by AsmaKacem1** [comment]
Hi @BatoolMM , I have finished the open-hardware.md 
could you please give me your opinion about it?

**Comment by ThanaMeri** [comment]
Marhba. @BatoolMM  I did some translations so far. it would be great if you give me a feedback

**Comment by nancyalaswad90** [comment]
Hi @BatoolMM , I have finished the open-access.md

 I wish you provide me with your feedback

**Comment by BatoolMM** [comment]
@nancyalaswad90 @AsmaKacem1 @ThanaMeri I added feedback to all of you in Crowdin - the next round of feedback will be given by the 9th of April.

**Comment by ThanaMeri** [comment]
@BatoolMM I didn't get any!

**Comment by BatoolMM** [comment]
@ThanaMeri - there are comments, just checked them now and tagged you. Let me know if you couldn't find them!

**Comment by nancyalaswad90** [comment]
Hi @BatoolMM , I have finished the open-access.md

Please  provide me with your feedback

**Comment by mejri1337** [comment]
Good evening @BatoolMM , if available , would you please check my translation of the overview-barriers.md file and provide me with feedback on what is approved and what needs rectification.

**Comment by BatoolMM** [comment]
@amj1337 @nancyalaswad90 I went through the translation and added a few comments. Please feel free to start the 2nd part by submitting a PR. I updated the instructions (above). If you have any questions, please let me know!

**Comment by aleesteele** [comment]
Hi, everyone! I just wanted to introduce myself in this issue, and say hello 👋! I'm Anne Lee Steele (@aleesteele), the Community Manager of The Turing Way. Together with @malvikasharan and @kirstieJane (who are co-leads of the project), we just wanted to say that we're so excited to follow the work you're doing with The Turing Way and with the amazing @BatoolMM! Translation is such important work that extends far beyond The Turing Way alone (as you all are well aware of - much moreso than me!)... and has much broader effects in the wider open source and open science communities. Best of luck with the application process, and if you have any questions about the project more broadly, please feel free to send me a message at asteele@turing.ac.uk. Congratulations again to you all! 🎉 🎉 🎉

**Comment by BatoolMM** [comment]
Thank you @aleesteele for introducing yourself, that's very thoughtful of you 🌷 .

I'd encourage everyone who has contributed so far in Crowdin to record their contribution in [Outreachy](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/) regardless if they completed their PRs or not. If you have any questions about how to record your contribution, please feel free to ask!

**Comment by nancyalaswad90** [comment]
> **Glad to know you  @aleesteele 🌷 .**


Hi @BatoolMM , really grateful for your support and here is my work  in previous two days 


- I check your comments and made changes to the open-access.md file.

- I just submitting a PR .

- And Finally add contribution in Outreachy.
.

![image](https://user-images.githubusercontent.com/36210723/163426129-1ea2bb77-223d-4f57-9e62-86d590ee32f0.png)
.

> **Please provide me with your feedback**

.

**Comment by BatoolMM** [comment]
Thank you to everyone who submitted a PR so far 🎉🎉 - Feedback is provided for PR #4 #5 #6!

**Comment by nancyalaswad90** [comment]
Hi @BatoolMM 


> **I approved all the suggestions feedback  and made the required change .**

.



**Thanks for your guidance , please review.**


.

**Comment by BatoolMM** [comment]
All PRs are merged, please don't forget to submit your final contribution through [Outreachy](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/). 

We have many strong candidates but we'll only take one intern so we would like to remind you to work on a minimum of 2-3 projects simultaneously so that they are not completely relying on their selection in OSCSA if we end up not selecting them!

If you have any questions, please feel free to ask!

**Comment by ThanaMeri** [comment]
@BatoolMM Do I still have time to post my PR ? I thought the deadline is April 22 ?

**Comment by BatoolMM** [comment]
@ThanaMeri of course, you are very welcome to submit your PR, the deadline is the 22nd of April!

**Comment by ThanaMeri** [comment]
@BatoolMM will you please give me a feedback on my PR ?

**Comment by BatoolMM** [comment]
All PRs are merged, and our newsletter mentions all your wonderful contributions. You can view the newsletter [here](https://mailchi.mp/ce1bff2f3963/open-science-community-saudi-arabia).  Please don't forget to record your final contributions in [Outreachy](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/). If you have any questions, don't hesitate to reach out!

**Comment by nancyalaswad90** [comment]
> All PRs are merged, and our newsletter mentions all your wonderful contributions. You can view the newsletter [here](https://mailchi.mp/ce1bff2f3963/open-science-community-saudi-arabia). Please don't forget to record your final contributions in [Outreachy](https://www.outreachy.org/outreachy-may-2022-internship-round/communities/open-science-community-saudi-arabia-oscsa/localisation-of-open-educational-resources-of-open/contributions/). If you have any questions, don't hesitate to reach out!
. 


**Thanks for your time and support** @BatoolMM 🌺🌺
.

**Comment by mejri1337** [comment]
Hi @BatoolMM I have just sent my PR, when available would you please check it out, thank you.

**Comment by BatoolMM** [comment]
Thank you for submitting PR #8 - please go through the comments I added!

**Comment by BatoolMM** [comment]
The intership has ended and this issue will be closed!
